### PR TITLE
Exposes a fast_fp8_accum option to _scaled_mm

### PR DESCRIPTION
**Pull request by drisspg**
# Summary
Adds the option to use fast_accumulation_mode for the fp8 matmul in scaled_mm

Information can be found here: https://docs.nvidia.com/cuda/cublas/#cublasltmatmuldescattributes-t
defaults to 0 (off)

cc @yanbing-j @vkuzo @albanD @kadeng

### CODE REVIEW

File: aten/src/ATen/cuda/CUDABlas.h [hunk 1]
```diff
@@ -115,13 +115,14 @@ void scaled_gemm(
     const void* mat2_scale_ptr,
     int64_t mat2_ld,
     ScalarType mat2_dtype,
-    const void* bias,
+    const void* bias_ptr,
     ScalarType bias_dtype,
     void* result_ptr,
-    const void* result_scale_ptr,
+    const void *result_scale_ptr,
```
Comment (ipiszy) [inline aten/src/ATen/cuda/CUDABlas.h:121]: nit: `void* ` to keep it consistent with lines above and below?

File: aten/src/ATen/cuda/CUDABlas.cpp [hunk 2]
```diff
@@ -897,17 +897,20 @@ void scaled_gemm(
     const void *result_scale_ptr,
     int64_t result_ld,
     ScalarType result_dtype,
-    void* amax_ptr) {
+    void* amax_ptr,
+    bool use_fast_accum) {
   #if CUDA_VERSION >= 11080
   const auto computeType = CUBLAS_COMPUTE_32F;
   const auto scaleType = CUDA_R_32F;
+  const int8_t fastAccuMode = (use_fast_accum) ? 1 : 0;
```
Comment (ipiszy) [inline aten/src/ATen/cuda/CUDABlas.cpp:905]: nit: `()` is not necessary?

**Review by ipiszy — APPROVED** [review]
Thanks @drisspg !

File: aten/src/ATen/cuda/CUDABlas.cpp (hunk 2 above)
Comment (drisspg) [reply aten/src/ATen/cuda/CUDABlas.cpp:905]: copy pasta, will remove

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**